test/integration/controller/job/job_controller_test.go

```diff
@@ -243,6 +243,17 @@ var _ = ginkgo.Describe("Job controller", func() {
 				createdWorkload.Status.Conditions[0].Status == metav1.ConditionTrue
 		}, util.Timeout, util.Interval).Should(gomega.BeTrue())
 	})
+	ginkgo.It("Should reconcile job when queueName set by annotation (deprecated)", func() {
```

Comment:
Thanks!

Add an empty line before `It`

Reply:
Done